subprojects/docs/docs.gradle

```diff
@@ -103,13 +103,45 @@ tasks.named("stageDocs") {
 samples {
     templates {
         javaAndroidApplication
-        springBootWebApplication
-        gradlePluginInJava
+        springBootWebApplication {
+            target = "app"
+        }
+        gradlePluginInJava {
+            target = "greeting-plugin"
+        }
         buildSrcPluginJavaModuleTransform
 
         javaApplication
         javaListLibrary
         javaUtilitiesLibrary
+        javaApplicationInApp {
```

Comment:
It looks like this might be the same thing as javaApplicationAsSubproject, just with a different subproject name.

There are other duplicates like this now.

Reply:
In general we need different ones, because sometimes everything is merged into one subproject and sometimes there are multiples. 

There might be some duplication if the name coincidentally is the same for one of the projects. But I think it is clearer to have it spelled out here.

I think we could improve this DSL such that you can define the target folders directly in the sample specification. Then we can get rid of this one indirection.